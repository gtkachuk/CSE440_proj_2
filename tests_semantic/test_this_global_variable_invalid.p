program testThisGlobalVariableInvalid;


class testThisGlobalVariableInvalid

BEGIN
   VAR
      this : integer;


FUNCTION testThisGlobalVariableInvalid;
VAR someVar : integer;
BEGIN
   this := 7
END

END
.
