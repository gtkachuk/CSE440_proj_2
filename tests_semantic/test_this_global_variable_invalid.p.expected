ERROR:15:Type mismatch between "testThisGlobalVariableInvalid" and "integer"
Errors detected. Exiting.
